Dockerfile

```diff
@@ -21,7 +21,4 @@ RUN if [[ "$PHP_VERSION" != 7.4* ]]; then \
     docker-php-ext-enable xdebug; \
     fi
 
-COPY composer.* ./
-RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
 COPY . .
```

Comment:
Ani to `COPY` neni potreba, cely zdrojak s mountuje pres volume

Reply:
@pepamartinec Ten `ci` container nemá namountěné volumes. 🤔 Takže přidat? A jestli jo, možná bych celej `ci` kontejner dal do hajzlu, protože ve flow se kvůli tomu spojenému cmd stejně volá `composer ci`.

Reply:
celej do hajzlu :slightly_smiling_face: V `platform-libraries` to funguje stejne tak (zadne `ci` veci https://github.com/keboola/platform-libraries/blob/c790b30a0637f53bccc1dcfd52e0c4021509ab8c/docker-compose.yml#L47)